### ISSUE TITLE
Don't depend on `cordova` command installed on host machine

### ISSUE THREAD

**Issue by mjsir911** [bhdouglass/clickable#52]
I'm trying to keep my developer machine clean, but clickable is requiring me to install the `cordova` package.

``` bash
$ clickable build
No clickable.json was found, using defaults and cli args
Auto detected template to be "cordova"
The command "cordova" does not exist on this system, please install it for clickable to work properly"
```

Is there any way to run all cordova commands inside the docker container? It does the compilation inside the container, so I'm not sure why it is needed on the host machine anyways.

**Comment by bhdouglass** [comment]
Cordova uses the old chroots, so that's why clickable depends on the cordova command being installed. If cordova was updated so it could be built in the clickable docker container then it would be simple to update clickable to use that.

**Comment by mjsir911** [comment]
fixed with #57
